lib/main_common.pm

```diff
@@ -2830,6 +2833,9 @@ sub load_ha_cluster_tests {
             loadtest 'sles4sap/netweaver_install';
             loadtest 'sles4sap/netweaver_cluster';
             loadtest 'sles4sap/sap_suse_cluster_connector';
```

Comment:
Maybe you should test but `sap_suse_cluster_connector` could (and I think should) be tested with HANA also, it's not a NW specific command. But maybe you can check (also possible to postpone this for another PR).

Reply:
I tried the sleep in the fencing module but boot_to_desktop fails.

Reply:
> I tried the sleep in the fencing module but boot_to_desktop fails.

This answer seems to not be in the right conversation :) I talked about `sap_suse_cluster_connector` here, plz add it after HANA cluster test.

Reply:
Ok.  Will add this in a later PR.